client/ayon_deadline/plugins/publish/global/collect_deadline_job_env_vars.py

```diff
@@ -53,3 +51,17 @@ def process(self, context):
 
         # Transfer some environment variables from current context
         context.data.setdefault(JOB_ENV_DATA_KEY, {}).update(env)
+
+
+class CollectAYONServerUrlToFarmJob(CollectDeadlineJobEnvVars):
+    label = "Submit AYON server URL to farm job"
+    settings_category = "deadline"
+
+    # Enable in settings
+    enabled = False
+
+    ENV_KEYS = [
+        "AYON_SERVER_URL",
+        "AYON_API_KEY",
+        "AYON_USE_DEV",
```

Comment:
Why do we need to pass this? AYON_BUNDLE_NAME covers that, or?

Reply:
That was my question - `AYON_USE_DEV` is hence redundant? We need the SERVER URL and API KEY to allow a single Deadline server to support submissions from multiple AYON servers (mostly for 'dev' usage currently).

Reply:
So this whole PR is adding new plugin and settings for dev?